packages/tuono/src/shared/TuonoScripts.tsx

```diff
@@ -1,18 +1,17 @@
 import type { JSX } from 'react'
 
-import { useRouterContext } from 'tuono-router'
-
 import { DevResources } from './DevResources'
 import { ProdResources } from './ProdResources'
+import { useTuonoContextServerPayload } from './TuonoContext'
 
 export function TuonoScripts(): JSX.Element {
-  const { serverSideProps } = useRouterContext()
+  const serverPayload = useTuonoContextServerPayload()
 
   return (
     <>
-      <script>{`window.__TUONO_SSR_PROPS__=${JSON.stringify(serverSideProps)}`}</script>
-      {serverSideProps?.mode === 'Dev' && <DevResources />}
-      {serverSideProps?.mode === 'Prod' && <ProdResources />}
+      <script>{`window.__TUONO_SERVER_PAYLOAD__=${JSON.stringify(serverPayload)}`}</script>
```

Comment:
How about turning this string into a constant and share it across the source?

```suggestion
      <script>{`window[payloadWindowLocation]=${JSON.stringify(serverPayload)}`}</script>
```

Reply:
`SERVER_PAYLOAD_VARIABLE_NAME` added